### PR TITLE
add option to ignore table

### PR DESCRIPTION
**Pull request by ming535**

### CODE REVIEW

File: docs/2.0/03-inputs-en.md [hunk 1]
```diff
@@ -114,19 +114,32 @@ location = "Local"
...
 schema = "test_2"
 table = "test_source_*"
+
 # Optional
 # Enforce the column to scan for tables. If you don't specifiy this value, the system will use a column that has unique index.
 # Make sure you know what you are doing: YOU NEED TO SPECIFY A COLUMN THAT HAVE A UNIQUE INDEX.
 scan-column = "id"
 
+# you can ignore talbes defined in input.config.table-configs
+# gravity do not support scan of these tables if it matches all of these condition:
```
Comment (Ryan-Git) [inline docs/2.0/03-inputs-en.md:140]: this statement is not related to `ignore-tables` config, it belongs to the whole mysql batch input.

File: docs/2.0/03-inputs-en.md [hunk 2]
```diff
@@ -114,19 +114,32 @@ location = "Local"
...
+#
+# 1. do not have primary key
+# 2. do not have unique key
+# 3. the number of rows in this table is greater than max-full-dump-count.
+#
+# To ignore these tables, you can setup a ingore table list.
+[[input.config.ignore-tables]]
```
Comment (Ryan-Git) [inline docs/2.0/03-inputs-en.md:147]: why not add a bool property in table config as before?

File: pkg/inputs/mysqlbatch/utils.go [hunk 3]
```diff
@@ -58,8 +58,12 @@ func GetTables(db *sql.DB, schemaStore schema_store.SchemaStore, tableConfigs []
 		}
 
 		for _, tableName := range allTables {
+			if ignoreTable(schemaName, tableName, ignoreTables) {
+				continue
+			}
+
 			for _, tablePattern := range tableConfigs[i].Table {
-				if utils.Glob(tablePattern, tableName) {
+				if tableConfigs[i].Schema == schemaName && utils.Glob(tablePattern, tableName) {
```
Comment (Ryan-Git) [inline pkg/inputs/mysqlbatch/utils.go]: also support blog for schema name?

File: docs/2.0/03-inputs-en.md (hunk 2 above)
Comment (ming535) [reply docs/2.0/03-inputs-en.md:147]: If we add it to TableConfig, it means []TableConfig will have a priority when selecting table. I think it is unclear when this priority combined with glob matching.

File: pkg/inputs/mysqlbatch/utils.go (hunk 3 above)
Comment (ming535) [reply pkg/inputs/mysqlbatch/utils.go]: maybe next version...

Comment (Ryan-Git) [inline pkg/inputs/mysqlbatch/utils.go]: schema is always equal here since it's already extracted from table config.

**Review by Ryan-Git — APPROVED** [review]
